### DISCUSSION TITLE
[Roadmap] v1.5

### DISCUSSION THREAD

**Discussion by AlexanderVolkovTCS** [Tinkoff/investAPI#447] [Announcements]
- Песочница: 
  - PositionStream в песочнице не всегда возвращал корректные total;
  - Приведение в соответствие GetSandboxOrderState и GetOrderState;
- Увеличено максимальное количество orderStream до 10;
- Выделен отдельный лимит на GetStopOrders();
- Исправлена проблема в параметре 1minCandle / 1DayCandle;
- Исправлена проблема в архиве исторических котировок;
- улучшен стриминг по вебсокетам:
  - повышена стабильность;
  - добавлены алиасы в параметрах;
  - опубликован простой GUI для вебсокетов;
- исправлена ошибка валидации UID в поле order_id методе PostOrder (проходил запрос без "-" );
- GetOperations: улучшено сопоставление разных инструментов одной позиции;
- Добавлена ошибка 30103 Для инструмента доступно выставление заявки только типа лучшая цена;
- Добавлена в FindInstrument фильтрация по опционам;
- Исправлена ошибка при запросе getLastPrices когда один из инструментов не найден;
- Разное название биржи (верхний  нижний кейс) в стримах и инструментах.

https://github.com/Tinkoff/investAPI/releases/tag/v1.5

**Comment by nonamegithub** [comment]
> * Выделен отдельный лимит на GetStopOrders();

Не подтверждаю

**Comment by ilyabrin** [comment]
@nonamegithub как я понял из коммента https://github.com/Tinkoff/investAPI/issues/446#issuecomment-1614771319, это публичные роадмапы на ближайшее время

@AlexanderVolkovTCS возможно, имеет смысл добавить слово Roadmap в заголовок и еще лейбл прописать

**Reply by nonamegithub** [reply]
Если так, то можно считать мой коммент ответом [на просьбу об обкатке и комментариях](https://github.com/Tinkoff/investAPI/issues/446#issuecomment-1614771319)
